os/mm/mm_heap/mm_manage_allocfail.c

```diff
@@ -99,6 +105,31 @@ void mm_manage_alloc_fail(struct mm_heap_s *heap, int startidx, int endidx, size
 	abort_mode = true;
 #endif
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
```

Comment:
How about adding a check on ```CONFIG_WATCHDOG``` also? I think ```CONFIG_WATCHDOG_DEVPATH``` is not defined when watchdog is not enabled and this might cause some errors.

Reply:
Thank you for review. I think so. added